ui/app/src/state/epics/content.ts

```diff
@@ -596,6 +596,9 @@ const content: CrafterCMSEpic[] = [
       filter(([{ payload }, state]) => Boolean(state.content.itemsByPath[payload.targetPath])),
       switchMap(([{ payload }, state]) =>
         fetchSandboxItemService(state.sites.active, payload.targetPath).pipe(
+          tap((item) => {
+            getHostToGuestBus().next(updateFieldValueOperationComplete({ item }));
```

Comment:
Don't think it is a good idea to use an action for something completely off its purpose.